universe/Tech.h

```diff
@@ -95,6 +95,9 @@ class FO_COMMON_API Tech {
     const std::vector<UnlockableItem>&    UnlockedItems() const
     { return m_unlocked_items; }
 
+    const ValueRef::ValueRef<double>* ResearchCost() const { return m_research_cost.get(); } //!< return value ref of research cost
```

Comment:
name as `ResearchCostRef` or similar to disambiguate with the same-name functions above

Reply:
Fixed